build.gradle.kts

```diff
@@ -3,20 +3,20 @@ version = "0.0.1"
 
 val jaxbVersion = "2.3.1"
 val kithApprecVersion = "2019.09.09-08-50-693492ddc1d3f98e70c1638c94dcb95a66036d12"
-val ktorVersion = "2.3.12"
+val ktorVersion = "3.0.2"
 val kluentVersion = "1.73"
 val mockkVersion = "1.13.12"
-val jacksonDataTypeVersion = "2.17.2"
+val jacksonDataTypeVersion = "2.18.0"
 val javaTimeAdapterVersion = "1.1.3"
-val jsonVersion = "20231013"
-val logbackVersion = "1.5.6"
+val jsonVersion = "20240303"
+val logbackVersion = "1.5.12"
 val logstashEncoderVersion = "7.4"
 val micrometerRegistryVersion = "1.12.8"
 val mqVersion = "9.4.0.0"
 val spekVersion = "2.0.19"
 val syfotjenesterVersion = "1.2021.06.09-13.09-b3d30de9996e"
 val swaggerUiVersion = "5.17.2"
-val kafkaVersion = "3.8.0"
+val kafkaVersion = "3.9.0"
 
 plugins {
     kotlin("jvm") version "1.9.24"
```

Comment:
Litt kjedelig, men det å bumpe denne til 2.0.21 brekker noen av testene - vanskelig å fikse dem (blir noe surr med mock'ene for Kafka-producer'ene).